spec/context_spec.rb

```diff
@@ -157,14 +157,20 @@
     expect(context.outcome).to eq(::LightService::Outcomes::SUCCESS)
   end
 
+  it "can contain false values" do
+    context = LightService::Context.make(:foo => false)
+    expect(context[:foo]).to eq false
+  end
+
   context "when aliases are included via .make" do
     let(:context) do
       LightService::Context.make(
         :foo => "foobar",
+        :foo2 => false,
```

Comment:
I think you meant `:bar2` here. When I run the specs as it is, it passes with the old code, but when I change this key to `:bar2`, it fails with receiving nil instead of `false`.